packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
   "author": "Celo",
   "license": "LGPL-3.0",
   "scripts": {
-    "lint:ts": "yarn run --top-level eslint .",
-    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
-    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "anvil-devchain:e2e-tests": "./scripts/foundry/run_e2e_tests_in_anvil.sh",
+    "anvil-devchain:integration-tests": "./scripts/foundry/run_integration_tests_in_anvil.sh",
```

Comment:
There is one big e2e test and two integration tests -> not sure if they are used that much, something to verify 😁

Reply:
I saw they are declared in foundry -> so assuming yes, resolving :)